x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/collection_gcp_pub_sub_subscription_creation.json

```diff
@@ -20,8 +20,9 @@
   ],
   "related_integrations": [
     {
+      "integration": "audit",
       "package": "gcp",
-      "version": "1.10.0"
+      "version": "^2.2.1"
```

Comment:
If we notice this, the reason is because before we were just doing the least compatible in general, instead we are doing the least compatible major version. The `^` is used by Kibana to determine anything with this version and above but only on minor and patch, not major which is why we filter to the latest major.